code/modules/clothing/under/color.dm

```diff
@@ -6,7 +6,7 @@
 
 /obj/item/clothing/under/color/random/Initialize()
 	..()
-	var/obj/item/clothing/under/color/C = pick(subtypesof(/obj/item/clothing/under/color) - /obj/item/clothing/under/color/random - /obj/item/clothing/under/color/grey/glorf - /obj/item/clothing/under/color/black/ghost)
+	var/obj/item/clothing/under/color/C = pick(subtypesof(/obj/item/clothing/under/color) - /obj/item/clothing/under/color/random - /obj/item/clothing/under/color/grey/glorf - /obj/item/clothing/under/color/black/ghost - /obj/item/clothing/under/color/black/skirt - /obj/item/clothing/under/color/grey/skirt - /obj/item/clothing/under/color/blue/skirt - /obj/item/clothing/under/color/green/skirt - /obj/item/clothing/under/color/orange/skirt - /obj/item/clothing/under/color/pink/skirt - /obj/item/clothing/under/color/red/skirt - /obj/item/clothing/under/color/white/skirt - /obj/item/clothing/under/color/yellow/skirt - /obj/item/clothing/under/color/darkblue/skirt - /obj/item/clothing/under/color/teal/skirt - /obj/item/clothing/under/color/lightpurple/skirt - /obj/item/clothing/under/color/darkgreen/skirt - /obj/item/clothing/under/color/lightbrown/skirt - /obj/item/clothing/under/color/brown/skirt - /obj/item/clothing/under/color/maroon/skirt)

```

Comment:
This here would leave me to believe there's an issue with how these items are typed.

Reply:
it has to do with rainbowtide. since its randomly picking an assistant colored uniform from the list, i had to exclude the skirts or else assistants would be spawning in them roundstart.

Reply:
Yes but look at _how_ you had to do it.

Basically you've typed it in a way that asks what color is the clothing BEFORE asking whether or not it's a jumpsuit or skirt. It should be flipped imo, which would turn this into a simple `- subtypesof(/obj/item/clothing/under/skirt)`

Reply:
Ah gotcha. Since all of the extra department outfits were subtypes, I made the skirts follow suit. I'll fix the assistant skirts so it looks like what you suggested. Makes way more sense now that I see what you're saying.